### DISCUSSION TITLE
Flowise and Jira Integration Query

### DISCUSSION THREAD

**Discussion by abhaysahu** [FlowiseAI/Flowise#3289] [Q&A]
Hello team,

I’m currently working on connecting Flowise with Jira to pull Jira stories for automation purposes. Does Flowise have any native functionality to access Jira directly, or would this need to be done through the Atlassian API? Any insights or guidance would be appreciated!

Thank You

**Comment by StoneToken** [comment]
Hi! As of flowise 3.0.1 there's a jira connector out of the box - https://docs.flowiseai.com/integrations/langchain/document-loaders/jira
